packages/client-db/src/Sync/worker_threads.d.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017-2018 @polkadot/client-db authors & contributors
```

Comment:
TODO: Move to https://github.com/polkadot-js/ts repo - easier here atm since it is still in flux